voxblox_tango_interface/test/tango_tsdf_to_esdf.cc

```diff
@@ -11,36 +11,55 @@ using namespace voxblox;  // NOLINT
 int main(int argc, char** argv) {
   google::InitGoogleLogging(argv[0]);
 
-  if (argc != 3) {
-    throw std::runtime_error("Args: filename to load, followed by filename to save to");
+  if (argc != 7) {
+    throw std::runtime_error(
+        std::string("Args: filename to load, filename to save to") +
+        ", min weight, min fixed distance" +
+        ", max esdf distance, default esdf distance");
   }
 
   const std::string file = argv[1];
+  FloatingPoint min_weight = std::stof(argv[3]);
+  FloatingPoint min_distance_m = std::stof(argv[4]);
+  FloatingPoint max_distance_m = std::stof(argv[5]);
+  FloatingPoint default_distance_m = std::stof(argv[6]);
 
   TangoLayerInterface::Ptr layer_from_file;
   io::TangoLoadLayer(file, &layer_from_file);
 
-  std::cout << "Layer memory size: " << layer_from_file->getMemorySize() << "\n";
+  // TODO(mereweth@jpl.nasa.gov) - can we get truncation distance from ntsdf
```

Comment:
This seems wrong, the block size is not necessarily the truncation size in tango AFAIK. Also the comment below does not really make sense here.

Reply:
I went back and reread this - better to just rely on the user to pass correct args.